tests/video/conftest.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: ai ts=4 sts=4 et sw=4 nu
+
+import pytest
+import pathlib
+
+from zimscraperlib.video import VidCompressionCfg, VidUtil
+
+
+@pytest.fixture(scope="function")
+def vidcompressioncfg():
+    return VidCompressionCfg()
+
+
+@pytest.fixture(scope="function")
+def vidutil():
+    return VidUtil(VidCompressionCfg(), "mp4", True)
+
+
+@pytest.fixture(scope="function")
+def temp_video_dir():
+    return pathlib.Path(__file__).parent.joinpath("temp")
+
+
+@pytest.fixture(scope="function")
+def hosted_video_links():
+    links = {
+        "mp4": "https://github.com/satyamtg/test-bucket/raw/master/video.mp4",
```

Comment:
👍 I see the files are not that big (700K and 1MB), we may consider adding them to the repo. not sure yet. Until that we'll move them to kiwix server.
I think it would make more sense to have test files with clear audio :) Even better maybe, a 1-2s long cut of an open video like big-buck-bunny or something from blender foundation?